src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +202,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+		processInputExecuteCommand();
```

Comment:
do take note of your indentation, whether is it space or tab. The coding standard says 4 spaces.